packages/frontend-2/components/projects/AddDialog.vue

```diff
@@ -60,7 +64,8 @@ const mp = useMixpanel()
 const onSubmit = handleSubmit(async (values) => {
   await createProject({
     ...values,
-    visibility: visibility.value
+    visibility: visibility.value,
+    workspaceId: props.workspaceId
```

Comment:
This correctly adds the project to the Workspace. `useCreateProject` correctly updates the cache for `projectsDashboardQuery`, but I couldn't get it to also update `workspacePageQuery`.

@fabis94 @Mikehrn If either of you have any suggestions here let me know. If it's easier to jump on a call, or to just do it yourself, feel free to do that too.

Reply:
I've updated the createProject mutation to handle cache updates for Workspace.projects as you suggested. I resolved some TypeScript and ESLint errors in the process but no sure everything was necessary. I'd appreciate any feedback on whether there's a better approach.